pkgs/build-support/testers/sourceEquivalence/tester.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, runCommand, dos2unix }:
+
+{ gitSource
+, src
+, testName ? "check-source-equivalence-${gitSource.name}-vs-${src.name}"
+}:
+
+runCommand testName {
+} ''
+  touch "$out"
+  mkdir -p tested/
+  tar xvf ${src} -C tested/ &>/dev/null
+  diff --brief --recursive --new-file --unified ${gitSource} tested/xz-5.6.1
```

Comment:
Why is there a hard-coded version (`xz-5.6.1`) here? `testers.sourceEquivalence` sounds like a generic tester, and it's a bit surprising to see a hard-coded version like this.

Reply:
I thought that it being a WIP would make it clear it's a WIP.

Reply:
Oops! Sorry for the noise.